include/openmc/particle_data.h

```diff
@@ -56,6 +56,23 @@ struct SourceSite {
   int64_t progeny_id;
 };
 
+//! State of a particle used for particle track files
+struct TrackState {
+  Position r;
+  Direction u;
+  double E;
+  double time {0.0};
+  double wgt {1.0};
+  int cell_id;
+  int material_id {-1};
```

Comment:
Is it worth putting in a comment here to explain why the material_id defaults to -1

Reply:
Definitely -- I've added a comment here